src/Microsoft.DocAsCode.Build.Engine/TemplateModelTransformer.cs

```diff
@@ -157,6 +157,27 @@ internal ManifestItem Transform(InternalManifestItem item)
             return manifestItem;
         }
 
+        private string GetLinkToPath(string fileName)
+        {
+            if (EnvironmentContext.FileAbstractLayerImpl == null)
+            {
+                return null;
+            }
+            var pp = ((FileAbstractLayer)EnvironmentContext.FileAbstractLayerImpl).GetOutputPhysicalPath(fileName);
+            var expandPP = Path.GetFullPath(Environment.ExpandEnvironmentVariables(pp));
+            var outputPath = Path.GetFullPath(_context.BuildOutputFolder);
+            if (expandPP.Length > outputPath.Length &&
+                FilePathComparer.OSPlatformSensitiveStringComparer.Equals(outputPath, expandPP.Remove(outputPath.Length)) &&
```

Comment:
Path.GetFullPath will normalize it.

---
In reply to: [102364867](https://github.com/dotnet/docfx/pull/1329#discussion_r102364867) [](ancestors = 102364867)